### PR TITLE
Added additional configuration options for dealing with AuthNContexts…

### PR DESCRIPTION
**Pull request by zmoore-usgs**
… and passive authentication.

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -9,9 +9,12 @@ security:
     key-store-saml-key: ${keystoreSAMLKey}
   saml:
     idp:
-      metadataLocation: ${samlIdpMetadataLocation}
+      metadata-location: ${samlIdpMetadataLocation}
```
Comment (isuftin) [inline src/main/resources/application.yml]: Is it possible to leave the other key in as well as to not break backwards compatibility and deprecate it?

Comment (isuftin) [reply src/main/resources/application.yml]: Otherwise we have to make sure that the next release of the Docker container for this makes this change

Comment (zmoore-usgs) [reply src/main/resources/application.yml]: I believe this should be backwards compatible as is. The key that changed is the internal key the application uses for this data. The key used to pass it in to the application, `samlIdpMetadataLocation` stayed the same.

Comment (isuftin) [reply src/main/resources/application.yml]: 👍

**Pull request merged**